packages/mds-compliance-service/repository/entities/compliance-array-response-entity.ts

```diff
@@ -0,0 +1,23 @@
+import { Entity, Column } from 'typeorm'
+import { IdentityColumn, RecordedColumn } from '@mds-core/mds-repository'
+import { ComplianceArrayResponseDomainModel } from '../../@types'
+
+export interface ComplianceArrayResponseEntityModel extends IdentityColumn, RecordedColumn {
+  compliance_array_response_id: ComplianceArrayResponseDomainModel['compliance_array_response_id']
+  compliance_snapshot_ids: string
```

Comment:
This type is probably wrong, I'd expect it to be a `UUID[]` (just an alias to `string[]`)

Reply:
Doesn't matter cause this type is getting nuked anyway, but I was storing it in the DB as a comma separated string. From what I'd seen in the Postgres docs, there wasn't a way to store string arrays, though I could certainly be mistaken.